llvm/test/Transforms/AlignmentFromAssumptions/domtree-crash.ll

```diff
@@ -0,0 +1,25 @@
+; RUN: opt -passes=alignment-from-assumptions -disable-output < %s
+
+; REQUIRES: asserts
```

Comment:
Drop this and add (generated) check lines, even if you're only testing for "does not crash".